db/migrate/20180523230926_create_rents.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rents do |t|
+      t.references :user, foreign_key: true, null: false, default: "user_id"
```

Comment:
Check the schema file, since a reference is of `bigint` type, this default resolves to a 0. Do we want this default?